homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,99 @@
+"""Support for Lutron lights."""
+import asyncio
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
```

Comment:
You can stay this part in sync as before but use https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/group.py#L339 for group.